canopycover.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python3
+"""Calculates canopy coverage for plots in georeferenced images
+"""
+
+import argparse
+import copy
+import json
+import logging
+import os
+from typing import Union
+import numpy as np
+import dateutil.parser
+from agpypeline import entrypoint, algorithm, geoimage
+from agpypeline.environment import Environment
+from osgeo import gdal, ogr
+
+from configuration import ConfigurationCanopycover
+
+
+# Array of trait names that should have array values associated with them
+TRAIT_NAME_ARRAY_VALUE = ['canopy_cover', 'site']
+
+# Mapping of default trait names to fixed values
+TRAIT_NAME_MAP = {
```

Comment:
we really need a better way of handling metadata ... but this doesn't need to hold up this PR

* access_level is a betydb specific thing. 
* species should be user provided (though 'unknown' is a sensible default
* author name etc should be in a codemeta.json file

Reply:
@dlbauer This appears to be a follow on of the conversation we had with Ken: https://hackmd.io/5kgZTUSIS1uNqvuFdvxPiQ?view.

There's a task outstanding for you to come up with the fields that should be in the CSV file